src/modeler/AutoRest.Swagger/Validation/TrackedResourcePatchOperationValidation.cs

```diff
@@ -26,6 +26,11 @@ public class TrackedResourcePatchOperationValidation : TypedRule<Dictionary<stri
         public override ValidationCategory ValidationCategory => ValidationCategory.RPCViolation;
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.SDKImpactingChanges;
```

Comment:
similarly as previous, I believe this would be service impacting if the patch operation is not implemented by the service.